tests/conftest.py

```diff
@@ -2038,7 +2038,7 @@ def core_dump_and_config_check(duthosts, tbinfo, request):
                         json.loads(duthost.shell("cat /etc/sonic/running_golden_config{}.json".format(asic_index),
                                                  verbose=False)['stdout'])
 
-    yield
+    yield duts_data
```

Comment:
We will add expected core dump files into `duts_data[duthost.hostname]["pre_core_dumps"]`, thus to avoid config reload caused by expected core dump files.